docker-entrypoint.sh

```diff
@@ -127,7 +127,12 @@ if [ -n "$isLikelyRedmine" ]; then
 			env="REDMINE_DB_${var^^}"
 			val="${!env}"
 			[ -n "$val" ] || continue
-			echo "  $var: \"$val\"" >> config/database.yml
+			if [ "$var" != 'adapter' ]; then
+				# https://github.com/docker-library/redmine/issues/353 🙃
+				val='"'"$val"'"'
```

Comment:
```suggestion
				val="\"$val\""
```

No strong opinion either way, just curious why not double quotes with escapes?

Reply:
Just that the highlight in my editor makes the former easier to see as "double quotes around a variable value" and `\` is ugly -- very superficial, and I'm happy to change it if you prefer or think it's easier for others the other way. :+1:

![image](https://github.com/user-attachments/assets/836a7d06-5be0-490d-90db-c229fe03bc33)

![image](https://github.com/user-attachments/assets/b38c7ef3-b857-4976-9e11-045377687da9)